src/gui/transferlistdelegate.cpp

```diff
@@ -58,15 +58,9 @@ void TransferListDelegate::paint(QPainter *painter, const QStyleOptionViewItem &
     newopt.minimum = 0;
     newopt.state = option.state;
     newopt.textVisible = true;
+    newopt.textAlignment = Qt::AlignCenter;
 
-#if defined(Q_OS_WIN) || defined(Q_OS_MACOS)
-    // XXX: To avoid having the progress text on the right of the bar
-    QProxyStyle fusionStyle {"fusion"};
-    QStyle *style = &fusionStyle;
-#else
-    QStyle *style = option.widget ? option.widget->style() : QApplication::style();
-#endif
-
+    const QStyle *style = option.widget ? option.widget->style() : QApplication::style();
```

Comment:
If we want to be totally const correct this should be `const QStyle *const style = blah`

Reply:
Is that really what you want?
I'm not that paranoid... So if you don't insist too much, I'd rather not go to extremes.

Reply:
I don't know :S 
But doesn't the same logic apply for when we do `const` for non-pointers?

Reply:
It is dangerous to have write access for any pointer, so we use `const int *a;`. And it is *relatively* safe to read from any pointer address, thus fixing a pointer address would be good but isn't strictly required (in my viewpoint).

Reply:
Yeah. Disregard my comment. I was reading the consts wrong. I forgot you essentially have to read them right-to-left in a pointer situation.